x-pack/auditbeat/processors/add_session_metadata/pkg/processdb/simple_test.go

```diff
@@ -0,0 +1,131 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package processdb
+
+import (
+	"testing"
+
+	"github.com/elastic/beats/v7/x-pack/auditbeat/processors/add_session_metadata/pkg/procfs"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/elastic/elastic-agent-libs/logp"
+
+	"golang.org/x/sys/unix"
+)
+
+var logger = logp.NewLogger("processdb")
+var reader = procfs.NewMockReader()
+
+// glue function to fit the return type required by these tests
+func newSimpleDBIntf(reader procfs.Reader) DB {
+	ret := NewSimpleDB(reader, *logger)
+	_ = ret.ScrapeProcfs()
+	return ret
+}
+
+func TestSimpleSingleProcessSessionLeaderEntryTypeTerminal(t *testing.T) {
```

Comment:
Suggest wrapping these up into a slice an using `t.Run` in a loop.

Reply:
This has all been removed, as the DB interface has been removed, and the tests in `entry_leader_test.go` are called directly